James Sutcliffe Test Task

JamesRobertSutcliffe opened the pull request:
Hi, 

Thanks for giving me the opportunity to take part in this task. I really enjoyed it and learned some new skills along the way. 

I took a mainly flexbox approach to building this task and tried to match the figma canvas as best a possible. This was my first time using SCSS so that was somewhat of a learning curve for me and had a impact on my utilisation of the mono package, although I tried to incorporate this where possible. 

If you have any questions please do not hesitate to reach out.

Many thanks,

James Sutcliffe
jsutcliffe1991@gmail.com
07507600229

JamesRobertSutcliffe committed:
Notes have been updated / research topics in notes before beginning code

JamesRobertSutcliffe committed:
Notes updated

JamesRobertSutcliffe committed:
CTA deciphered / move onto research

JamesRobertSutcliffe committed:
npm start to run site in parcel

JamesRobertSutcliffe committed:
SCSS components built

JamesRobertSutcliffe committed:
Card layour developed further

JamesRobertSutcliffe committed:
Zoom feature developed

JamesRobertSutcliffe committed:
Further progress made / sort black underlay for text over image

JamesRobertSutcliffe committed:
Drop Shadow fix developed / fine tune text colours and sort location icon next

JamesRobertSutcliffe committed:
Cards developed / move onto CTA next

JamesRobertSutcliffe committed:
First phase complete

JamesRobertSutcliffe committed:
Initial commit

JamesRobertSutcliffe committed:
Media queries added / basic responsiveness developed / CTA basic boxes also developed / Add extra medium media query? Or push out to 900px?

JamesRobertSutcliffe committed:
MVP developed / see notes for finalisation (add extra media query / add comments)

JamesRobertSutcliffe committed:
Comments added and ready for PR

bbucek commented:
CTA:
- No margin below
- Looks strange on mobile 
- Text chevron is not looking great

Cards:
- Height is strange
- No backdrop under text, although some text shadow to help with legibility

JamesRobertSutcliffe committed:
Solution for black underlay beginning to be developed

JamesRobertSutcliffe committed:
Background gradient added / address other comments next

JamesRobertSutcliffe committed:
Backgrounds added / see readme for toDos - next address height feedback

JamesRobertSutcliffe committed:
Height sizes developed

JamesRobertSutcliffe committed:
2 points to address in read me / all feedback dealt with / just icons and ipad breakpoint to be sorted

JamesRobertSutcliffe committed:
link to icon site added to read me / see notes there

JamesRobertSutcliffe committed:
All feedback addressed / Tablet breakpoint added

JamesRobertSutcliffe commented:
Hi, 

Thank you for the feedback, I have now addressed this feedback:

**CTA:**

-     Margin bottom has been added
-     Aesthetics on phone have been improved 
-     Text chevron has been improved

**Cards:**

- Strange height issue has been fixed
- Card gradient underlay added

![Screenshot 2023-05-31 170629](https://github.com/primate-inc/test-task/assets/107635428/722e7bae-e6bf-4781-a1ca-10625d2ea4ed)

I have also added a break point for tablets so site is now responsive between phone, tablet and laptops.

Please reach out with any more feedback or Queries. 

Many thanks,

James Sutcliffe
jsutcliffe1991@gmail.com
07507600229
